internal/cli/cmd/grant/grant.go

```diff
@@ -161,9 +161,20 @@ func (o *grantOptions) run(ctx context.Context) error {
 	} else {
 		req.Request = &svcv1alpha1.GrantRequest_UserClaims{
 			UserClaims: &rbacapi.UserClaims{
-				Subs:   o.Subs,
-				Emails: o.Emails,
-				Groups: o.Groups,
+				Claims: []rbacapi.Claim{
+					{
+						Name:   "subs",
+						Values: o.Subs,
+					},
+					{
+						Name:   "emails",
+						Values: o.Emails,
+					},
+					{
+						Name:   "groups",
+						Values: o.Groups,
+					},
+				},
```

Comment:
Same situation as described [here](https://github.com/akuity/kargo/pull/2315/files#r1692023211)

Reply:
Added that flag as above